api-test/hasura/actions/_handlers/insertOrgMember.test.ts

```diff
@@ -34,7 +34,7 @@ beforeAll(async () => {
 });
 
 test('Check that the created user has been added to org_members', async () => {
-  await new Promise(res => setTimeout(res, 1000));
+  await new Promise(res => setTimeout(res, 2000));
```

Comment:
this kind of wait should be unnecessary... if it passes with a delay and fails without it, it's an indication that a promise / async function call is not being awaited properly